source/features/user-local-time.tsx

```diff
@@ -109,6 +109,8 @@ async function insertUserLocalTime(hovercardContainer: Element): Promise<void> {
 	// Adding the time element might change the height of the hovercard and thus break its positioning
 	const hovercardHeight = hovercard.offsetHeight;
 
+	// Only remove the reserved space when the time element is actually inserted in the hovercard to avoid #4527
+	hovercardContainer.classList.add('rgh-user-local-time-added');
```

Comment:
The idea is to use two classes, one for de-duplication (added as soon as possible), the other to remove the space reserved via CSS (added as late as possible).